src/platform_impl/windows/event_loop.rs

```diff
@@ -730,18 +750,19 @@ unsafe fn flush_paint_messages<T: 'static>(
                 return;
             }
 
-            if 0 == winuser::PeekMessageW(
+            if !PeekMessageW(
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)